web/modules/custom/dbcdk_community/src/Form/ProfileEditForm.php

```diff
@@ -153,10 +149,10 @@ public function buildForm(array $form, FormStateInterface $form_state, $username
     ];
 
     $form['birthday'] = [
-      '#type' => 'date',
+      '#type' => 'datetime',
```

Comment:
It's not necessarily better, it's just cleaner since we don't have to use a [class constant](https://github.com/DBCDK/biblo-admin/pull/47/files#diff-3fb04a6aef57ae4c52468bf184cacdebL27)